### PR TITLE
added support for malaysia language

### PR DESCRIPTION
**Pull request by kusumandaru**
Add malaysia support language, same language have similarity with indonesia, except few words for nil and eight

### CODE REVIEW

**Comment by styd** [comment]
Hi, @kusumandaru. You don't seem to be a malaysian. Have you consult malaysian people about the translations?

**Comment by kusumandaru** [comment]
WhatsApp Image 2019-10-16 at 16 07 58

already confirm with malaysian people, he confirm okay

**Comment by kusumandaru** [comment]
[numbering.docx](https://github.com/radar/humanize/files/3733908/numbering.docx)

File: spec/locales/ms_spec.rb [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    [44_333_221_111_112_176_514_321_007_310, "empat puluh empat oktil tiga ratus tiga puluh tiga septillion dua ratus dua puluh satu sextillion seratus sebelas quintillion seratus dua belas quadrillion seratus tujuh puluh enam trilion lima ratus empat belas bilion tiga ratus dua puluh satu juta tujuh ribu tiga ratus sepuluh"],
+    [444_333_221_111_112_176_514_321_007_310, "empat ratus empat puluh empat oktil tiga ratus tiga puluh tiga septillion dua ratus dua puluh satu sextillion seratus sebelas quintillion seratus dua belas quadrillion seratus tujuh puluh enam trilion lima ratus empat belas bilion tiga ratus dua puluh satu juta tujuh ribu tiga ratus sepuluh"],
+    [5_444_333_221_111_112_176_514_321_007_310, "lima empat ratus empat puluh empat oktil tiga ratus tiga puluh tiga septillion dua ratus dua puluh satu sextillion seratus sebelas quintillion seratus dua belas quadrillion seratus tujuh puluh enam trilion lima ratus empat belas bilion tiga ratus dua puluh satu juta tujuh ribu tiga ratus sepuluh"],
```
Comment (styd) [inline spec/locales/ms_spec.rb]: Your LOTS is upto `oktil`. Starting from one grouping above that should be unreadable.

File: lib/humanize.rb [hunk 2]
```diff
@@ -30,7 +30,7 @@ def humanize(locale: Humanize.config.default_locale,
   def self.for_locale(locale)
     case locale.to_sym
     # NOTE: add locales here in ealphabetical order
-    when :az, :de, :en, :es, :fr, :id, :pt, :ru
+    when :az, :de, :en, :es, :fr, :id, :pt, :ru, :ms
```
Comment (styd) [inline lib/humanize.rb]: Existing seems to be sorted alphabetically.

File: spec/locales/ms_spec.rb (hunk 1 above)
Comment (kusumandaru) [reply spec/locales/ms_spec.rb]: highest count is octillion, is there any way to grouping above oktil
or may i remove the spec?

File: lib/humanize.rb (hunk 2 above)
Comment (kusumandaru) [reply lib/humanize.rb]: on it

File: spec/locales/ms_spec.rb (hunk 1 above)
Comment (styd) [reply spec/locales/ms_spec.rb]: Yes. You should remove all unreadable numbers from test. That's the limitation of `ms` language.

Comment (kusumandaru) [reply spec/locales/ms_spec.rb]: done

File: lib/humanize.rb (hunk 2 above)
Comment (kusumandaru) [reply lib/humanize.rb]: done

File: lib/humanize/locales.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 module Humanize
-  %w[az de en es fr id pt ru th tr].each do |locale|
+  %w[az de en es fr id pt ru th tr ms].each do |locale|
```
[the remark this answers is not in the document]
Comment (kusumandaru) [reply lib/humanize/locales.rb]: done

**Pull request merged**